[2025-04-11T05:57:01.312+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.load_to_sqlite manual__2025-04-11T05:50:45.806125+00:00 [queued]>
[2025-04-11T05:57:01.322+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.load_to_sqlite manual__2025-04-11T05:50:45.806125+00:00 [queued]>
[2025-04-11T05:57:01.322+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-11T05:57:01.359+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_sqlite> on 2025-04-11 05:50:45.806125+00:00
[2025-04-11T05:57:01.366+0000] {standard_task_runner.py:60} INFO - Started process 203 to run task
[2025-04-11T05:57:01.369+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_modo_prueba', 'load_to_sqlite', 'manual__2025-04-11T05:50:45.806125+00:00', '--job-id', '439', '--raw', '--subdir', 'DAGS_FOLDER/etl_workflow.py', '--cfg-path', '/tmp/tmpviqaqym4']
[2025-04-11T05:57:01.370+0000] {standard_task_runner.py:88} INFO - Job 439: Subtask load_to_sqlite
[2025-04-11T05:57:01.443+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_modo_prueba.load_to_sqlite manual__2025-04-11T05:50:45.806125+00:00 [running]> on host bdba95992caa
[2025-04-11T05:57:01.600+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='simon' AIRFLOW_CTX_DAG_ID='etl_pipeline_modo_prueba' AIRFLOW_CTX_TASK_ID='load_to_sqlite' AIRFLOW_CTX_EXECUTION_DATE='2025-04-11T05:50:45.806125+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-11T05:50:45.806125+00:00'
[2025-04-11T05:57:01.601+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-11T05:57:01.602+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/load_to_sqlite.py']
[2025-04-11T05:57:01.613+0000] {subprocess.py:86} INFO - Output:
[2025-04-11T05:57:02.683+0000] {subprocess.py:93} INFO - âœ… Archivo parquet leÃ­do: /opt/***/data/processed/final_merged.parquet
[2025-04-11T05:57:02.684+0000] {subprocess.py:93} INFO - ðŸ”— Conectado a la base de datos SQLite en /opt/***/data/output/merged_data.db
[2025-04-11T05:57:02.685+0000] {subprocess.py:93} INFO - ðŸ“¦ Datos cargados en la tabla 'datos_finales'
[2025-04-11T05:57:02.685+0000] {subprocess.py:93} INFO - âœ… ConexiÃ³n cerrada. Proceso de carga completado.
[2025-04-11T05:57:02.765+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-11T05:57:02.830+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_modo_prueba, task_id=load_to_sqlite, execution_date=20250411T055045, start_date=20250411T055701, end_date=20250411T055702
[2025-04-11T05:57:02.904+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-11T05:57:02.925+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
